nbconvert/exporters/html.py

```diff
@@ -89,8 +140,9 @@ def _file_extension_default(self):
     def _template_name_default(self):
         return 'lab'
 
-    theme = Unicode('light',
-                    help='Template specific theme(e.g. the JupyterLab CSS theme for the lab template)'
+    theme = Unicode(
+        'light',
+        help="the JupyterLab CSS theme for the lab template"
```

Comment:
Maybe we should also mention somewhere that only JupyterLab themes distributed as prebuilt extensions will be compatible?

Reply:
Indeed! I shouldn't have changed this comment that way though, I will revert this change and specify that it supports prebuilt extensions